<launch>
    <arg name="rosbag_record" default="false"/>
    <arg name="bag_path" default="Desktop"/>
    <arg name="bag_name_prefix" default="tare_garage"/>
    <arg name="rviz" default="true"/>
    <!-- 是否自定义探索边界 -->
    <arg name="use_boundary" default="false"/>

    <include file="$(find explorer)/launch/explore.launch">
        <arg name="scenario" value="garage"/>
        <arg name="rosbag_record" value="$(arg rosbag_record)"/>
        <arg name="bag_path" value="$(arg bag_path)"/>
        <arg name="bag_name_prefix" value="$(arg bag_name_prefix)"/>
        <arg name="rviz" value="$(arg rviz)"/>
    </include>

    <group if="$(arg use_boundary)">
        <node pkg="explorer" type="navigationBoundary" name="navigationBoundary" output="screen" required="true">
            <param name="boundary_file_dir" type="string" value="$(find explorer)/data/boundary.ply"/>
            <param name="sendBoundary" type="bool" value="true"/>
            <param name="sendBoundaryInterval" type="int" value="2"/>
        </node>
    </group>
</launch>
